packages/patternlab/styleguide/source/assets/js/modules/mainNavHamburger.js

```diff
@@ -969,6 +967,12 @@ if (hamburgerMenuContainer) {
       "iframe.goog-te-menu-frame.skiptranslate"
     );
 
+    // Hide "+" symbol from screen readers.
+    let hamburgerGTSymbol = utilNarrowNav.querySelector(".ma__utility-nav__translate #google_translate_element a span");
+    if (hamburgerGTSymbol) {
+      hamburgerGTSymbol.setAttribute("aria-hidden", "true");
```

Comment:
```suggestion
      hamburgerGTSymbol.setAttribute("aria-hidden", "true");
```

Is this for mobile? On mobile, I'm still hearing the "plus" being announced next to "select language". I think it's because the + sign is added in css content. You can try this in css https://www.w3.org/TR/css-content-3/#accessibility to set the alt text to empty

Reply:
Yes, it's for mobile.
First, I thought it was because of pseudo content, but all the top menus of the navigation and the log in to button have the "+" as pseudo content of `button > span.toggle-indicator`.  Their symbols don't get announced.

In the current code, the the symbol is added to `#google_translate_element a`.  To add `aria-hidden="true"`, I added `span` to the `a` as its child element.  This new structure matches to the top menus and the log in to button in the flyout that has the "+" symbol.  Only difference is they are `button`s as GT is `a`.
<img width="436" alt="Mass_Gov_and__header-mixed_scss_—_mayflower" src="https://github.com/massgov/mayflower/assets/9633303/eff6f6e9-f5c3-4aa2-be3a-41d950cee660">

With the testing on Nov. 22, It's confirmed that JAWS, NVDA and VO don't announce the symbol.

Reply:
![Screen Shot 2023-11-22 at 12 40 13 PM](https://github.com/massgov/mayflower/assets/5789411/75ee9dd6-c9c3-4111-aabd-253311912ba5)

Weird, I'm still hearing the plus...

Reply:
ok so I gather that the UAT link is not always reliable https://mayflower.digital.mass.gov/patternlab/b/patternlab/DP-24290_navigation-labels/?p=pages-Homepage

I pulled this down locally and tested it again, now the phrase select language is not being announced, because it's wrapped inside of the aria-hidden span
![Screen Shot 2023-11-22 at 12 48 15 PM](https://github.com/massgov/mayflower/assets/5789411/dc5983a1-4664-414e-9d30-b68a08e64dc8)

Reply:
As moving the `:after` from `a` to `a > span`, the pseudo si  allied to all `span`s under `a`.  This was causing  this issue although strangely, all screen readers announced "select language" as I tested.

To resolve this, targeted the last `span` under `a` to have `aria-hidden` and the pseudo-class to add the symbol.

Reply:
`aria-label` is added to the UL. It's confirmed that JAWS and NVDA announced it.